src/ch/ehi/ili2db/gui/Config.java

```diff
@@ -789,4 +790,12 @@ public boolean isDisableRounding() {
     public void setDisableRounding(boolean disableRounding) {
         this.disableRounding = disableRounding;
     }
+
+	public boolean isXtfAttrText() {
```

Comment:
wie SQL_NULL implementieren, damit die Einstellung in die t_ili2db_settings Tabelle gespeichert wird (und der Benutzer somit beim --import/--export nicht mehr wissen/angeben muss)